vivisect/__init__.py

```diff
@@ -1796,7 +1796,10 @@ def addMemoryMap(self, va, perms, fname, bytes):
         self._fireEvent(VWE_ADDMMAP, (va, perms, fname, bytes))
 
     def delMemoryMap(self, va):
-        raise "OMG"
+        '''
+        Remove a memory map from the workspace.
```

Comment:
likewise, lets document what `va` means

Reply:
same deal, i'll make this `mapva` or do you think `baseva` would be more descriptive?
i originally had the logic such that any va in the map would delete it, but that didn't seem right.... not at all.

Reply:
no real preference, both make the intent more clear. thanks!

Reply:
updated.